bundle/config/mutator/translate_paths.go

```diff
@@ -91,6 +92,9 @@ func (t *translateContext) rewritePath(
 	input string,
 	opts translateOptions,
 ) (string, error) {
+	// If the input is a local requirements file, we need to translate it to an absolute path.
+	input, _ = libraries.IsLocalRequirementsFile(input)
```

Comment:
This needs to conditionally assign `input` based on the `ok` value.

Reply:
Done https://github.com/databricks/cli/pull/2736/commits/ee64e50560029a499a95fb4a4646c3a6811bdcae